pennylane/qchem/molecule.py

```diff
@@ -72,8 +72,20 @@ def __init__(
         normalize=True,
     ):
 
-        if basis_name not in ["sto-3g", "STO-3G", "6-31g", "6-31G"]:
-            raise ValueError("Currently, the only supported basis sets are 'sto-3g' and '6-31g'.")
+        if basis_name not in [
```

Comment:
Same comment as above. Something like this could be done.
```suggestion
        if basis_name.lower() not in ["sto-3g", "6-31g", "6-311g", "cc-pvdz"]
```
Additionally, you might have forgotten to reflect these new additional basis sets in the Molecule class docstring.

Reply:
Please see my comment above.